src/main/java/org/killbill/billing/invoice/api/InvoiceUserApi.java

```diff
@@ -357,6 +364,7 @@ public InvoiceItem insertInvoiceItemAdjustment(UUID accountId, UUID invoiceId, U
      * @param context the tenant context
      * @throws InvoiceApiException
      */
+    @RequiresPermissions(INVOICE_CAN_DELETE_CBA)
```

Comment:
Perhaps, it would make more sense to have `HA` permission for these kind if things